packages/terra-form-checkbox/src/Checkbox.jsx

```diff
@@ -171,12 +167,10 @@ const Checkbox = ({
 
   return (
     <div className={checkboxClasses}>
-      <div className={selectContainer}>
```

Comment:
The `selectionContainer` div is redundant - the label itself can serve the same functionality, providing an inline-selection-hitbox for both the checkbox and label.